package.json

```diff
@@ -64,18 +64,11 @@
     }
   ],
   "jspm": {
-    "directories": {
-      "lib": "dist"
-    },
     "main": "dist/goldenlayout",
-    "files": [
-      "dist/goldenlayout.js",
-      "dist/goldenlayout.min.js"
-    ],
     "format": "global",
     "registry": "jspm",
     "dependencies": {
-      "jquery": "*"
+      "jquery": "^2.1.0"
```

Comment:
GoldenLayout supports IE 8 - which isn't supported by jQuery 2.* . Could we leave the jQuery version open?